playbooks/technical-planning.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Technology choices
+description: How to write technical plans at Artsy
+---
+
+# Technical planning
+
+Technical planning is a crucial part of our engineering process. Writing tech plans helps us:
+- Keep track of technical decisions that we've made, and the associated rationale.
+- Streamline a team's development of a feature and align within and across teams.
+- Communicate and collaborate effectively on technical decisions in an async manner.
+- Think through problems in a way that allows developers to practice technical decision-making & grow.
+- Get feedback early on in the development process, to avoid costly U-turns further down the line.
+
+Developers should use their judgment when deciding whether or not to write a tech plan, but we recommend them when planning:
+1. Technical design for medium to large features
+1. Architectural changes or new design patterns
+1. Changes affecting other teams' products or engineers' workflows
+1. New integrations or services
+
+If the feature you're working on follows our best-practices and doesn't require technical decisions larger than what would fit in a single PR, it's OK to skip.
+
+If you're working on a small feature or it doesn't fit the criteria above but you still want to write one, that's fine! Feel free to keep it small.
+
+## Tech Plans vs. Product Briefs
+Technical plans discuss and propose a technical implementation, while product briefs should govern the design, user experience, and business requirements of a feature. We recommend linking to an associated product brief for context, but product and requirement decisions should live on the product brief, not on the tech plan.
+
+It's natural to discover new product implications through the tech planning process. If this happens, make sure those discussions are routed to the product brief for clarity.
+
+If product-related feedback comes up while the tech plan is being reviewed, that is often a sign that there is more to discuss and debate at the product brief-level. If you are authoring a tech plan and this happens, it is recommended that you bring the conversation immediately to the associated PM(s). Once the question or feedback is addressed, the tech planning can continue.
+
+## Tech Plan States
+### Draft
+A tech plan in this state is a work in progress.
+
+### Proposed
+A tech plan in this state is officially open for feedback. Anyone should feel free to look at Proposed tech plans and leave their thoughts.
+
+### Accepted
```

Comment:
**suggestion**: Would we consider technical plan states that aren't so interconnected with stakeholder review states?

Inspired by the interaction between GitHub Pull Requests and Review Requests, we could have something like this:

## Technical Plans

```mermaid
flowchart LR
  Draft <-- ready for feedback --> Open
  Draft --> Closed
  Open -- acceptance criteria met --> Implementing
  Open --> Closed
  Implementing -- executed --> Done
  Implementing ---> Closed
  style Draft fill:#FFF689
  style Open fill:#7CA5B8
  style Closed fill:#EC0B43
  style Implementing fill:#FFA400
  style Done fill:#84B082
```

## Review Requests

```mermaid
flowchart LR
  Pending <--> Approved
  Pending <--> ChangesRequested(Changes Requested)
  style Pending fill:#7CA5B8
  style ChangesRequested fill:#EC0B43
  style Approved fill:#84B082
```

Reply:
Ooh I like this. So we remove the "Approved" state in favor of "Implementing".